### ISSUE TITLE
Revert stealing

### ISSUE THREAD

**Issue by mo8it** [qu1x/bevy_trackball#1]
I want to disable the trackball while dragging objects around and then enable it when they are dropped. I tried using `set_stolen(true)` which indeed disables the trackball, but `set_stolen(false)` doesn't re-enable it.

Am I supposed to use something other than `set_stolen` to temporarily disable the trackball?

Thanks for your awesome crate!

**Comment by n3vu0r** [comment]
Hi, the `set_stolen` method was intended to be called every frame even if its argument hasn't changed. It uses a counter to delay the release by another frame, which is necessary for some mouse events captured by `egui` on `wasm` if I remember correctly.

Just calling it twice should work for you as a quick fix:

```rust
// disable trackball
viewport.set_stolen(true);

// enable trackball
viewport.set_stolen(false);
viewport.set_stolen(false);
```

This should probably be redesigned for the next release, maybe by moving the counter into a resource of the `egui_viewport_theft` system.

**Comment by mo8it** [comment]
Funny hack, but I will try it for now :P

**Comment by n3vu0r** [comment]
Refactored [`set_stolen`](https://qu1x.github.io/bevy_trackball/doc/bevy_trackball/struct.TrackballViewport.html#method.set_stolen). Also fixed mouse and touch input issues introduced with Bevy `0.12.1` when giving back viewport. Just released `0.3.0`, let me if it works for you.

**Comment by mo8it** [comment]
It works, thanks!

But calling the argument an option of the number of "frames" is very confusing. These aren't frames unless this method is called every frame 😅
